### PR TITLE
fix: Responsiveness for the code table

### PR DESCRIPTION
**Pull request by itzabhinavarya**
## What does this PR do?

This PR Fixes the responsiveness for the small devices, the demo installation of the code section is not showing fully.
image

Fixes # (issue)

image

-->

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### CODE REVIEW

File: apps/formbricks-com/components/home/Steps.tsx [hunk 1]
```diff
@@ -21,9 +21,10 @@ export const Steps: React.FC = () => {
         heading="Set Formbricks up in minutes"
         subheading="Formbricks is designed for as little dev attention as possible. Here’s how:"
       />
-      <div id="howitworks" className="mx-auto mb-12 mt-16 max-w-lg md:mb-0 md:mt-8 md:max-w-none">
+      <div id="howitworks" className="m-auto mb-12 mt-16 max-w-lg md:mb-0 md:mt-8 md:max-w-none">
         <div className="px-4 sm:max-w-4xl sm:px-6 lg:max-w-7xl lg:px-8">
           <div className="grid md:grid-cols-2 md:items-center md:gap-16">
+            {/* <div className="inline-flex flex-col items-center"> */}
```
Comment (ShubhamPalriwala) [inline apps/formbricks-com/components/home/Steps.tsx:27]: small nit: Can you remove this additonal commented line please?

**Review by ShubhamPalriwala — COMMENTED** [review]
Thanks a lot for the quick fix! Just a small nit about removing an additional line and then we can merge this.


+50 points for solving it! 🔥

File: apps/formbricks-com/components/home/Steps.tsx (hunk 1 above)
Comment (itzabhinavarya) [reply apps/formbricks-com/components/home/Steps.tsx:27]: Yeah! Sure. 
Just removing right now.

**Review by ShubhamPalriwala — APPROVED** [review]

**Comment by itzabhinavarya** [comment]
Hii @ShubhamPalriwala  Sir, I have done with the removing line as you said. Please review the changes.

**Review by ShubhamPalriwala — APPROVED** [review]
LGTM! 🚀

**Comment by itzabhinavarya** [comment]
@ShubhamPalriwala That's great to me, Must assign this task to me as well sir, Haha..xD

**Comment by itzabhinavarya** [comment]
Thank you @ShubhamPalriwala Sir, for merging the PR.

**Pull request merged**